src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -584,6 +539,50 @@ export default class ResultsViewPage extends React.Component<
         });
     }
 
+    readonly userMessages = remoteData({
```

Comment:
lets just add a comment here saying "this is here temporarily for orienting user to expression data in plots page" or something.